templates/bluemix/manifest.yml

```diff
@@ -0,0 +1,16 @@
+declared-services:
+  cloudant-db:
+    label: cloudantNoSQLDB
+    plan: Lite
+applications:
+- path: .
+  memory: 256M
```

Comment:
I am developing optional prompts for those values with current values as defaults.